tests/Test+Shape+Estimation.ipynb

```diff
@@ -0,0 +1,1225 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import galsim\n",
+    "from om10 import DB\n",
+    "import os\n",
+    "import pandas as pd\n",
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "import sys\n",
+    "realizer_path = os.path.join(os.environ['SLREALIZERDIR'], 'slrealizer')\n",
+    "sys.path.insert(0, realizer_path)\n",
+    "from realize_om10 import OM10Realizer\n",
+    "from utils.utils import *\n",
+    "from utils.constants import *\n",
+    "#from realize_sl import SLRealizer\n",
+    "\n",
+    "%matplotlib inline\n",
+    "%load_ext autoreload\n",
+    "%autoreload 2"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Includes:\n",
+    "# 1. Determining nx, ny, and pixel scale for GalSim's drawImage and \n",
+    "# 2. Comparing the true and emulated images\n",
+    "# 3. Comparing the HSM output with the analytically derived properties"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 1. Checking that all functions of OM10Realizer run as expected"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "I write this test in a notebook because I'll need to look at images for each (nx, ny, pixel scale) configuration and qualitatively judge whether it will be okay for HSM shape estimation. First, I define the filepath for the test lens catalog and test observation history. I created these files just for testing purposes; they only have one row each."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "data_path = os.path.join(os.environ['SLREALIZERDIR'], 'data')\n",
+    "\n",
+    "test_catalog_f = os.path.join(data_path, 'test_catalog.fits')\n",
+    "observation_f = os.path.join(data_path, 'twinkles_observation_history.csv')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "f = os.path.join(data_path, 'qso_mock.fits')\n",
+    "from astropy.io import fits\n",
+    "hdul = fits.open(f)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "d = hdul[1].data\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 20,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [
```

Comment:
Notebooks should not have outputs in them!

Reply:
Oops I missed this one, sorry! Cleared output